app/scripts/components/granules/overview.js

```diff
@@ -115,10 +118,20 @@ class GranulesOverview extends React.Component {
     return <Overview items={overview} inflight={false} />;
   }
 
+  csvDownloadSection (fileData) {
+    if (fileData) {
+      const data = new Blob([fileData], {type: 'text/csv'});
+      const url = window.URL.createObjectURL(data);
+
+      return (<a className='csv__download' id='download_link' download='granules.csv' href={url}>Download Granule List</a>);
```

Comment:
The ticket mentions integration tests. Is the plan to add a Cypress test that shows at least that the link exists when CSV data comes back from the API? It would also be interesting to test actually clicking the download link to see if you get the right behavior, but that may be more challenging